eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -151,14 +157,54 @@ stages:
                   deploy:
                     steps:
                       - checkout: self
+                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
+                        parameters:
+                          SourceFolder: ${{parameters.ArtifactName}}-signed
+                          TargetFolder: ${{artifact.safeName}}
+                          FileFilter: ${{artifact.name}}.[0-9]*.[0-9]*.[0-9]*
```

Comment:
Should we make this value the default filter in the template?

Reply:
We could, but it I'm pretty confident that it'd be a whole nother can of worms to attempt to resolve the inevitable wonky yml rendering issues.